config/routes.rb

```diff
@@ -63,6 +63,8 @@
   end
 
   resources :polls, concerns: :statusable do
+    post 'make_choise', on: :member
```

Comment:
More semantic HTTP would be just `POST /polls/:id/choice` and `GET /polls/ready`.
So the HTTP endpoints are things, items, nouns, not action verbs.

This also will make your scary route helpers nice: `ready_index_polls_path` will become `ready_polls_path`, which reads almost like a valid sentence.